src/Components/profileOutline.js

```diff
@@ -0,0 +1,208 @@
+// code here! 
+// beep boop beep boop
+import PropTypes from 'prop-types';
+
+import { useEffect } from "react";
+
+const styles = {
+    header: {
+        fontSize: 69,
+        color: "maroon",
+        fontWeight: 900,
+    }, 
+    text: {
+        fontSize: 69,
+        color: "brown",
+    },
+};
+
+
+export default function ProfileOutline({
+    name, 
+    title,
+    pronouns,
+    email,
+    phone,
+    city,
+    state,
+    ethnicity,
+    age,
+    genderIdentity,
+    sexuality,
+    veteran,
+    disability,
+    housingSituation,
+    employmentStatus,
+    priorConvictions,
+}
+){
+// some code here 
+// const user = {
+//     name: 'kaylee',
+//     title: 'administrator',
+//     pronouns: 'she/her',
+//     email: 'kaeleytran@gmail.com', 
+// };
+
+useEffect(()=> {
+// do something 
+},//set something 
+);
+
+return (
+<div>
+    <div style={styles.header}>
```

Comment:
There is also a lot of repetition here! I think some ways we can clean this up are use classNames. That way we can apply the same css styles to multiple elements. Also, we can use nested elements for different pieces of information rather than an individual div for each element. Lmk if you want more guidance on this and I can give you references!